addons/hr/models/hr_employee_base.py

```diff
@@ -33,7 +33,7 @@ class HrEmployeeBase(models.AbstractModel):
         store=True,
         readonly=False,
         check_company=True)
-    work_phone = fields.Char('Work Phone', compute="_compute_phones", store=True, readonly=False)
+    work_phone = fields.Char('Work Phone', compute="_compute_phones", store=True, readonly=False, inverse='_inverse_work_contact_details')
```

Comment:
Doesn't this compute also have to be adjusted? `work_phone` is computed using `employee.address_id.phone` but then the 'inverse sets' `employee.work_contact_id.phone`